apps/web/src/pages/TalentNominations/NominateTalent/components/Details.tsx

```diff
@@ -1,43 +1,655 @@
+                  options?.talentNominationLateralMovementOptions,
+                  intl,
+                  [
+                    TalentNominationLateralMovementOption.SmallDepartment,
+                    TalentNominationLateralMovementOption.LargeDepartment,
+                    TalentNominationLateralMovementOption.CentralDepartment,
+                    TalentNominationLateralMovementOption.NewDepartment,
+                    TalentNominationLateralMovementOption.ProgramExperience,
+                    TalentNominationLateralMovementOption.PolicyExperience,
+                    TalentNominationLateralMovementOption.Other,
+                  ],
+                )}
+              />
+              {lateralMovementOptionOther && (
+                <Input
+                  type="text"
+                  id="lateralMovementOptionsOther"
+                  name="lateralMovementOptionsOther"
+                  label={intl.formatMessage({
+                    defaultMessage: "Other lateral move option",
+                    id: "BNSbyC",
+                    description:
+                      "Label other lateral move option input on the details step",
+                  })}
+                  rules={{
+                    required: intl.formatMessage(errorMessages.required),
+                  }}
+                />
+              )}
+            </div>
+          )}
+          {developmentProgram && (
+            <div
+              data-h2-display="base(flex)"
+              data-h2-flex-direction="base(column)"
+              data-h2-gap="base(x1)"
+            >
+              <div>
+                <p data-h2-font-weight="base(700)">
+                  {intl.formatMessage({
+                    defaultMessage: "Development program options",
+                    id: "PZEAt8",
+                    description:
+                      "Title for development program options section in nominations details step",
+                  })}
+                </p>
+                <p>
+                  {intl.formatMessage({
+                    defaultMessage:
+                      "Indicate one or more movement options that are recommended for the nominee.",
+                    id: "oiYxBW",
+                    description:
+                      "Description for development program options section in nominations details step",
+                  })}
+                </p>
+              </div>
+              <Checklist
+                idPrefix="developmentPrograms"
+                name="developmentPrograms"
+                legend={intl.formatMessage({
+                  defaultMessage: "Development program options",
+                  id: "OTI+Kb",
+                  description:
+                    "Label for the development program options checklist on the details step",
+                })}
+                rules={{
+                  required: intl.formatMessage(errorMessages.required),
+                }}
+                items={[
```

Comment:
This should only be development programs for the specific community. And eventually, not even all those: #12981 .  I'd recommend pulling them from developmentInterests property on the Community model.  If you do want you stick will pulling the whole list, make sure you filter them by community before showing the list.

Reply:
Oops, you're right. It should be the set of development programs associated directly with the talent nomination event. 👍  I'm going to close #12981, too.

Reply:
Sounds good! Fixed in [1dbefdf](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13111/commits/1dbefdf47158728cdeec25b784fef1d5bd8ea689)